content/blog/2018/10/2018-10-15-custom-war-packager.adoc

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: "Build your own Jenkins! Introducing Custom WAR/Docker Packager"
+tags:
+- tools
+- docker
+- jenkins-x
+- cloud-native
+author: oleg_nenashev
+---
+
+I would like to introduce link:https://github.com/jenkinsci/custom-war-packager[Custom WAR Packager] -
+a new tool for Jenkins administrators and developers.
+This tool allows packaging custom Jenkins distributions as WAR files,
+link:https://github.com/jenkinsci/docker[Docker images]
+and link:https://github.com/jenkinsci/jenkinsfile-runner[Jenkinsfile Runner] bundles.
+This tools allows packaging Jenkins, plugins, and configurations in a ready-to-fly distribution.
+Custom WAR packager is a part of the Ephemeral Jenkins master toolchain
+which we presented in our link:/blog/2018/09/12/speaker-blog-a-cloud-native-jenkins/[A Cloud Native Jenkins] blogpost.
+This toolchain is already used in link:https://jenkins-x.io[Jenkins X] to package single-shot masters.
+
+In this blogpost I will show some common use-cases for Custom WAR Packager.
+
+== History
+
+As Jenkins itself, Custom WAR Packager has started as a small development tool in April 2018. 
```

Comment:
"As __with__ Jenkins itself, Custom WAR Packager ~~has~~ started as a small development tool in April 2018." (Although this is still in danger of making it sound like Jenkins started in April 2018 so may be worth splitting in to two sentences.)